tests/GoogleAuthenticatorTest.php

```diff
@@ -68,6 +68,63 @@ public function testCheckCode($expectation, $inputDate): void
         );
     }
 
+    /**
+     * @dataProvider testCheckCodeDiscrepancyData
+     */
+    public function testCheckCodeDiscrepancy($expectation, $inputDate): void
+    {
+        $authenticator = new GoogleAuthenticator(6, 10, new \DateTime('2012-03-17 22:17:00'), 30);
+
+        try {
+            $datetime = new \DateTime($inputDate);
+        } catch (\Exception $e) {
+            return;
+        }
```

Comment:
What is this try/catch for?

Reply:
The one existing test had used a try/catch around creating the DateTime object, so I just followed the existing test. I think the idea was to test invalid dates, but currently its just returning out of the test, so maybe I can change it to actually test the invalid date through an assertion.

Reply:
I'm not even sure there should be a test for the invalid date, because in that case the system under test is `\DateTime`, right?

Reply:
Yes that is true, I'll remove that test case.